addons/minio/template/base/install.sh

```diff
@@ -8,12 +8,31 @@ function minio_pre_init() {
     if [ -z "$MINIO_CLAIM_SIZE" ]; then
         MINIO_CLAIM_SIZE="10Gi"
     fi
+
+    # verify if we need to migrate away from the deprecated 'fs' format.
+    local minio_replicas
+    minio_replicas=$(kubectl get --no-headers deploy minio -n "$MINIO_NAMESPACE" -o custom-columns=:.spec.replicas 2>/dev/null)
+    if [ -n "$minio_replicas" ] && [ "$minio_replicas" != "0" ] && minio_uses_fs_format ; then
+        printf "${YELLOW}\n"
+        printf "The installer has detected that the cluster is running a version of minio backed by the now legacy FS format.\n"
+        printf "To be able to upgrade to the new Minio version a migration will be necessary. During this migration, Minio\n"
+        printf "will be unavailable.\n"
+        printf "\n"
+        printf "For further information please check https://github.com/minio/minio/releases/tag/RELEASE.2022-10-29T06-21-33Z\n"
+        printf "${NC}\n"
+        printf "Would you like to proceed with the migration ?"
+        if ! confirmN; then
+            bail "Not migrating"
```

Comment:
Should we allow people to continue through the rest of the install process even if they skip upgrading minio?

Reply:
(if we do, make sure to run `minio_object_store_output`)

Reply:
it would have to skip the minio upgrade

Reply:
as agreed we are going to block the upgrade.